src/OpenAi.php

```diff
@@ -4,8 +4,8 @@
 
 class OpenAi
 {
-    private string $engine = "davinci";
-    private array $headers;
+    private $engine = "ada";
```

Comment:
Davinci should stay for best experience 😊

```suggestion
    private string $engine = "davinci";
```